src/v2/Components/NavBar/NavBar.tsx

```diff
@@ -243,7 +244,17 @@ export const NavBar: React.FC = track(
                 </NavItem>
               </NavSection>
               <NavSection alignItems="right" mr={2}>
-                <NavItem href="#download-app-banner">Download App</NavItem>
+                <ScrollIntoView
+                  selector="#download-app-banner"
+                  behavior="smooth"
+                >
```

Comment:
I remembered this after pairing -- whenever we want something to jump to part of the page, we can wrap it in this component, and if we want to to animate we can use the `behavior='smooth'` prop